wiki/en/Tips-Tricks-More.md

```diff
@@ -80,21 +80,92 @@ Here is the script:
  jack_connect Jamulus:'output right' system:playback_2
 ~~~
 
-### Using ctrlmidich for MIDI controllers
+### Using `--ctrlmidich` for MIDI controllers
 
-The volume fader, pan control and mute and solo buttons in the Client's mixer window strips can be controlled using a MIDI controller by using the `--ctrlmidich` parameter (note: only available for use with macOS and Linux using Jamulus version 3.7.0 or higher, and on Windows using the Jamulus version with JACK support). To enable this feature, Jamulus must be launched with `--ctrlmidich`. There is one global MIDI channel parameter (1-16) and two parameters you can set for each item controlled: `offset` and `consecutive CC numbers`. Set the first parameter to the channel you want Jamulus to listen on (0 for all channels) and then specify the items you want to control (f = volume fader; p = pan; m = mute; s = solo; o = mute myself) with the offset (CC number to start from) and number of consecutive CC numbers. There is one exception that does not require establishing consecutive CC numbers which is the "Mute Myself" command - it only requires a single CC number as it is only applied to one's own audio stream. Take the following example:
+When this option is used on the command line, Jamulus will display a Server channel number assigned to each Client, which can be used to control the channel using MIDI CC numbers.  (Note: This is only available for use with macOS and Linux using Jamulus version 3.7.0 or higher, and on Windows using the Jamulus version with JACK support.  It also relies on the Jamulus UI being displayed.)
 
-`--ctrlmidich "1;f0*8;p16*8;s32*8;m48*8;o64"`
+Make sure you connect your MIDI device's output port to the Jamulus MIDI in port (QjackCtl (Linux/Windows), MIDI Studio (macOS) or whatever you use for managing connections). In Linux you will need to install and launch a2jmidid so your device shows up in the MIDI tab in Qjackctl.
```

Comment:
Is it really called MIDI studio on macOS?
I thought it's Audio/MIDI setup. @gilgongo

Reply:
I think I wrote this section back in the day. No idea where MIDI Studio came from. A quick check confirms that it's "Audio MIDI Setup". Also, I see there's a closing bracket missing in `(QjackCtl (Linux/Windows)`, or it needs to be rephrased.

Reply:
`(QjackCtl (Linux/Windows), MIDI Studio (macOS) or whatever you use for managing connections)` looks balanced.

Reply:
> Is it really called MIDI studio on macOS? I thought it's Audio/MIDI setup. @gilgongo

I'm afraid I no longer have access to a Mac, but Apple talk about [Audio/MIDI setup here](https://support.apple.com/en-gb/guide/audio-midi-setup/ams59f301fda/mac ).

Reply:
> (QjackCtl (Linux/Windows), MIDI Studio (macOS) or whatever you use for managing connections) looks balanced.

Oops, wasn't looking at the bigger picture!

Reply:
But yes, I'll fix the macOS audio/midi thing.  (I thought it was always "Audio/MIDI setup"? At least it was about 14 years ago when I last saw a mac.)

Reply:
Done.